From 27e8e6ac3bbbbd3536e8245d9ee6ca32c1fb196a Mon Sep 17 00:00:00 2001
From: Vernon Yang <vernon2gm@gmail.com>
Date: Wed, 11 Jan 2023 21:53:48 +0800
Subject: [PATCH 32/41] maple_tree: fix comment of mte_destroy_walk

The parameter name of maple tree is mt, make the comment to be mt
instead of mn, and the separator between the parameter name and
the description to be : instead of -.

Fixes: 54a611b60590 ("Maple Tree: add new data structure")
Signed-off-by: Vernon Yang <vernon2gm@gmail.com>
---
 lib/maple_tree.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index 77f146b8a..7065ce29b 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -5579,8 +5579,8 @@ static void mt_destroy_walk(struct maple_enode *enode, unsigned char ma_flags,
 
 /*
  * mte_destroy_walk() - Free a tree or sub-tree.
- * @enode - the encoded maple node (maple_enode) to start
- * @mn - the tree to free - needed for node types.
+ * @enode: the encoded maple node (maple_enode) to start
+ * @mt: the tree to free - needed for node types.
  *
  * Must hold the write lock.
  */
-- 
2.39.0.rc2.1.gbd5df96b79

